docs/channels.md

```
@@ -152,6 +152,18 @@ Agent Session の LLM 呼び出し時に、Channel Log の直近 30 件を一時
 | ボット応答 | 保存 | 保存 |
 | Single-Agent チャネル | — | 保存（従来通り） |
 
+#### `agent_send` メッセージ表示
+
+`agent_send` ツールによるエージェント間メッセージは、チャネルに次の形式で表示される:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)